app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/QuestionnaireFragment.java

```diff
@@ -0,0 +1,85 @@
+package io.github.droidkaigi.confsched2017.view.fragment;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import javax.inject.Inject;
+
+import io.github.droidkaigi.confsched2017.databinding.FragmentQuestionnaireBinding;
+import io.github.droidkaigi.confsched2017.model.Questionnaire;
+import io.github.droidkaigi.confsched2017.viewmodel.QuestionnaireViewModel;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.schedulers.Schedulers;
+
+public class QuestionnaireFragment extends BaseFragment implements QuestionnaireViewModel.Callback {
```

Comment:
Currently, we don't need to use `Callback` class.
Instead of `Callback`, please use `ResourceResolver` and `Navigator` in ViewModel.